index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="icon" href="src/assets/icons/tk.jpg">
+        <title>CV Tukach Kirill</title>
+        <link rel="stylesheet" href="style.scss">
+    </head>
+
+    <body>
+        <header class="header">
+            <a href="#" class="logo">logo</a>
+            <div class="container header-container">
+                <nav class="nav">
+                    <ul class="nav-list">
+                        <li class="nav-item">
+                            <a href="#my-contacts" class="nav-link">My contacts</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#about-me" class="nav-link">About me</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#my-skills" class="nav-link">My skills</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#education" class="nav-link">Education</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#experience" class="nav-link">Experience</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#languages" class="nav-link">languages</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main class="main">
+            <div class="container main-container">
+
+                <section class="section-left section-profile">
+                    <article class="section-profile_avatar">
+                        <img src="src/assets/photos/photo.jpg" alt="avatar" width="200" class="profile-avatar">
+                    </article>
+                    <article class="section-profile_title">
+                        <h1 class="profile-name">Tukach Kirill</h1>
+                        <h5 class="profile-location">Belarus, Gomel</h5>
+                        <h3 class="profile-speciality">robotics engineer</h3>
+                    </article>
+                </section>
+
+                <section class="section-left section-contacts" id="my-contacts">
+                    <h2 class="section-title">My contacts</h2>
+                    <ul class="section-list">
+                        <li class="list_items">
+                            <div class="icons contacts-phone"></div>
```

Comment:
It looks like you're going to use this <div> for the icon. If that's the case, then fine. But it's better to write code as needed. It is not being used in any way now.